+siibra/+core/Parcellation.m

```diff
@@ -1,32 +1,32 @@
 classdef Parcellation < handle
     properties
-        Id
-        Name
-        Atlas
-        Modality
-        Description
-        Graph
-        Spaces
+        id
```

Comment:
It's actually more common for MATLAB classes to use UpperCamel for property names